Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation3D.cxx

```diff
@@ -288,10 +288,16 @@ int vtkSlicerAngleRepresentation3D::RenderTranslucentPolygonalGeometry(
   count = this->Superclass::RenderTranslucentPolygonalGeometry(viewport);
   if (this->LineActor->GetVisibility())
     {
+    // The internal actor needs to share property keys. This allows depth peeling
+    // etc to work.
```

Comment:
` This allows depth peeling etc to work.` -> `This ensures the mapper states is consistent and allows depth peeling to work as expected.`

Reply:
Updated.